test_scripts/Defects/4_5/commonDefects.lua

```diff
@@ -0,0 +1,301 @@
+---------------------------------------------------------------------------------------------------
+-- RC common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.ValidateSchema = false
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonSteps = require("user_modules/shared_testcases/commonSteps")
+local commonTestCases = require("user_modules/shared_testcases/commonTestCases")
+local mobile_session = require("mobile_session")
+local json = require("modules/json")
+local events = require("events")
+
+--[[ Local Variables ]]
+local ptu_table = {}
+local hmiAppIds = {}
+
+local commonDefect = {}
+
+commonDefect.timeout = 2000
+commonDefect.minTimeout = 500
+commonDefect.DEFAULT = "Default"
+commonDefect.buttons = { climate = "FAN_UP", radio = "VOLUME_UP" }
+
+local function getPTUFromPTS(tbl)
+  tbl.policy_table.consumer_friendly_messages.messages = nil
+  tbl.policy_table.device_data = nil
+  tbl.policy_table.module_meta = nil
+  tbl.policy_table.usage_and_error_counts = nil
+  tbl.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
+  tbl.policy_table.module_config.preloaded_pt = nil
+  tbl.policy_table.module_config.preloaded_date = nil
+end
+
+function commonDefect.DefaultStruct()
+  return {
+    keep_context = false,
+    steal_focus = false,
+    priority = "NONE",
+    default_hmi = "NONE",
+    groups = { "Base-4"},
+  }
+end
+
+local function updatePTU(tbl)
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] = commonDefect.DefaultStruct()
+end
+
+local function jsonFileToTable(file_name)
+  local f = io.open(file_name, "r")
+  local content = f:read("*all")
+  f:close()
+  return json.decode(content)
+end
+
+local function tableToJsonFile(tbl, file_name)
+  local f = io.open(file_name, "w")
```

Comment:
@GetmanetsIrina May be it is worth to add check that file was successfully opened ?